frontend/public/components/cluster-settings/cluster-operator.tsx

```diff
@@ -60,9 +60,9 @@ const OperatorStatusIconAndLabel: React.FC<OperatorStatusIconAndLabelProps> = ({
 
 const tableColumnClasses = [
   '',
-  '',
-  'pf-m-hidden pf-m-visible-on-md',
-  'pf-m-hidden pf-m-visible-on-md',
+  'pf-u-w-16-on-xl',
+  'pf-u-w-33-on-2xl pf-m-hidden pf-m-visible-on-md',
```

Comment:
It seems like in the other instances you are doing : `hidden visible size`.  Would be good to be consistent.